src/Microsoft.TestPlatform.Extensions.BlameDataCollector/HangDumperFactory.cs

```diff
@@ -31,18 +31,19 @@ public IHangDumper Create(string targetFramework)
                 return new NetClientDumper();
             }
 
-            // this is not supported yet
-            // if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
-            // {
-
-            // if (frameworkVersion != default && frameworkVersion <= new Version("5.0"))
-            // {
-            //    return new SigtrapDumper();
-            // }
-
-            // EqtTrace.Info($"HangDumperFactory: This is OSX on netcoreapp3.1 or newer, returning the standard NETClient library dumper.");
-            // return new NetClientDumper();
-            // }
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                if (!string.IsNullOrWhiteSpace(targetFramework) && !targetFramework.Contains("v5.0"))
```

Comment:
Are newer frameworks e.g. 5.1, 5.2... or 6 etc. explicitly unsupported?

Reply:
Yeah that one needs improvement. Added to work list here https://github.com/microsoft/vstest/issues/2380